x-pack/legacy/plugins/ml/common/util/validators.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/**
+ * Provides a validator function for maximum allowed input length.
+ * @param maxLength Maximum length allowed.
+ */
+export function maxLengthValidator(
```

Comment:
This function is now used as falsy check, the contents of the returned object are ignored. 
Maybe it should be changed to just return true or false.

Reply:
Actually, i just realised that there is a `string_utils.js` file along side this file. IMO, this function is worth moving inside there, as it is validating the length of a string.

Reply:
I use it as falsy check because of the current code structure. Validator function provides useful info to render error messages as well as error key which lets generate an object with all errors for specific field/model.